BuildTasks/Common/Common.ts

```diff
@@ -79,7 +79,25 @@ export function setTfxManifestArguments(tfx: ToolRunner): (() => void) {
         const isPreview = extensionVisibility.indexOf("preview") >= 0;
 
         jsonOverrides.public = isPublic;
-        if (isPreview) { jsonOverrides.galleryFlags = ["Preview"]; };
+
+        if (isPreview) {
+            jsonOverrides.galleryFlags = jsonOverrides.galleryFlags || [];
```

Comment:
Agreed. Replaced the line below with .Push. 
Kept the coalesce in there to make the code aligned with the pricing option below and to make sure it keeps working if the order is changed.